sdk-core/src/main/java/care/data4life/sdk/network/model/RecordEncryptionService.kt

```diff
@@ -0,0 +1,331 @@
+/*
+ * Copyright (c) 2021 D4L data4life gGmbH / All rights reserved.
+ *
+ * D4L owns all legal rights, title and interest in and to the Software Development Kit ("SDK"),
+ * including any intellectual property rights that subsist in the SDK.
+ *
+ * The SDK and its documentation may be accessed and used for viewing/review purposes only.
+ * Any usage of the SDK for other purposes, including usage for the development of
+ * applications/third-party applications shall require the conclusion of a license agreement
+ * between you and D4L.
+ *
+ * If you are interested in licensing the SDK for your own applications/third-party
+ * applications and/or if you’d like to contribute to the development of the SDK, please
+ * contact D4L by email to help@data4life.care.
+ */
+
+package care.data4life.sdk.network.model
+
+import care.data4life.crypto.GCKey
+import care.data4life.crypto.KeyType
+import care.data4life.sdk.ApiService
+import care.data4life.sdk.crypto.CryptoContract
+import care.data4life.sdk.data.DataResource
+import care.data4life.sdk.fhir.Fhir3Resource
+import care.data4life.sdk.fhir.Fhir4Resource
+import care.data4life.sdk.fhir.FhirContract
+import care.data4life.sdk.lang.CoreRuntimeException
+import care.data4life.sdk.lang.DataValidationException
+import care.data4life.sdk.model.ModelContract
+import care.data4life.sdk.model.ModelContract.ModelVersion.Companion.CURRENT
+import care.data4life.sdk.tag.Annotations
+import care.data4life.sdk.tag.TaggingContract
+import care.data4life.sdk.tag.Tags
+import care.data4life.sdk.wrapper.WrapperContract
+
+class RecordEncryptionService(
```

Comment:
is final class a thing in Kotlin?

Reply:
By default, all classes in Kotlin are final.